tests/e2e/cucumber/features/smoke/shares/share.feature

```diff
@@ -158,3 +158,44 @@ Feature: share
 
     And "Brian" navigates to the shared with me page
     And "Brian" logs out
+
+
+  Scenario: receive two shares with same name
+    Given "Admin" creates following users using API
+      | id    |
+      | Carol |
+    And "Alice" logs in
+    And "Alice" creates the following folder in personal space using API
+      | name        |
+      | test-folder |
+    And "Alice" creates the following files into personal space using API
+      | pathToFile   | content      |
+      | testfile.txt | example text |
+    And "Alice" opens the "files" app
+    And "Alice" shares the following resource using the sidebar panel
+      | resource     | recipient | type  | role     |
+      | testfile.txt | Brian     | user  | Can view |
+      | test-folder  | Brian     | user  | Can view |
+    And "Alice" logs out
+    And "Carol" logs in
+    And "Carol" creates the following folder in personal space using API
+      | name        |
+      | test-folder |
+    And "Carol" creates the following files into personal space using API
+      | pathToFile   | content      |
+      | testfile.txt | example text |
+    And "Carol" opens the "files" app
+    And "Carol" shares the following resource using the sidebar panel
+      | resource     | recipient | type | role     |
+      | testfile.txt | Brian     | user | Can view |
+      | test-folder  | Brian     | user | Can view |
+    And "Carol" logs out
+    When "Brian" navigates to the shared with me page
+    Then following resources should be displayed in the Shares for user "Brian"
+      | resource         |
+      | testfile.txt     |
+      | testfile (1).txt |
+    # https://github.com/owncloud/ocis/issues/8471
+    #  | test-folder      |
+    #  | test-folder (1)  |
```

Comment:
issue is only for folder shares?

Reply:
I see that works
<img width="1022" alt="image" src="https://github.com/owncloud/web/assets/84779829/4f616a76-62ab-4791-8d1e-f02c15213da7">

Reply:
The issue might occur for files as well as in this issue https://github.com/owncloud/ocis/issues/8471.

Reply:
then will the test get flaky if we keep `testfile (1).txt`?

Reply:
In this scenario, the shares list for the user is:
```
testfile.txt
testfile (1).txt
test-folder
test-folder (2)
```

While i did it manually, the shares list for the user is:
```
testfile.txt
testfile (3).txt
test-folder
test-folder (1)
```
Not so sure why we get different results while testing manually and in automation.